qiskit_cold_atom/synqs_fermions/__init__.py

```diff
@@ -0,0 +1,9 @@
+# Gate imports
+from .fqs_gate_library import (
```

Comment:
We should use the existing gates as much as possible and if something is lacking from those we should extend the library.